src/status_im/constants.cljs

```diff
@@ -609,3 +609,12 @@
 
 (def ^:const min-token-decimals-to-display 6)
 (def ^:const swap-proposal-refresh-interval-ms 15000)
+
+(def ^:const wallet-transaction-estimation
+  {:unknown                 0
+   :less_than_one_minute    1
+   :less_than_three_minutes 2
+   :less_than_five_minutes  3
+   :more_than_five_minutes  4})
```

Comment:
I used snake_case as In constants file i didn't see any kebab-case.  shouldn't we use the same style in constants file? @clauxx

Reply:
I think we use snake case for strings that come from external sources e.g JSON-RPCs like `eth_sendTransaction` and we hide them behind a constant, so the project is still consistent with the naming convention. Here we have keywords with underscores though, so it could be that we will use e.g. ` :less_than_five_minutes` outside the constants file, which will not be consistent. Since we already do the conversion from string to keyword somewhere, might as well convert it to kebab case as well.

Reply:
Just to clarify, the estimation values are received on the mobile side as integers. I created this constant to match the same enum and also to serve as documentation. Therefore, using kebab-case is not necessary in this case IMO.

Reply:
> I used snake_case as In constants file i didn't see any kebab-case

> Therefore, using kebab-case is not necessary in this case IMO.

I'm confused. AFAIK kebab-case is the convention for clojure and I couldn't find any examples of snake_case

Reply:
Sorry if my explanation is not clear enough, the point is we could directly use input from rpc in the format function , but i though number alone can confuse us in future, so i just created the same enum as go side has. 
everything in constants file is snake_case. am i missed something? 
without constant it looks like this: 
```clojure
(defn estimated-time-format
  "Formats the estimated time for a transaction"
  [estimated-time]
  (condp = estimated-time
    1              "> 5"
    2              "< 1"
    3              "1-3"
    4              "3-5"
    "> 5"))
```

Reply:
I understand that what we use is the number and agree that using keywords makes it more understandable. My point is only about changing e.g. `:less_than_one_minute` (snake case) to `:less-than-one-minute` (kebab case)

> everything in constants file is snake_case 

We have no variable names or keywords that use snake case in the constants namespace or the project (from what I saw), and if there are then it's likely there by mistake

Reply:
Oh my bad, sorry i so confused with kebab case 😓, yes that's right it should be kebab-case, i will fix it.